bokehjs/gulp/tasks/test.coffee

```diff
@@ -39,6 +50,9 @@ mocha = (options={}) ->
       process.on('SIGINT',  handler)
   )
 
+gulp.task "test:coverage", ["defaults:generate"], () ->
+  gulp.src(["./test/all.coffee"]).pipe(mocha({coverage: true}))
+
 gulp.task "test", ["defaults:generate"], () ->
   gulp.src(["./test/all.coffee"]).pipe(mocha())
```

Comment:
it's not generated by `gulp test` by design (at least for now) you have to run `gulp test:coverage` explicitly. Some day it would be nice to run on Travis and upload the reports, for now I just want something that can be generated locally when desired.